samples/openapi3/client/petstore/python/tests_manual/test_issue10155.py

```diff
@@ -0,0 +1,61 @@
+# coding: utf-8
+
+"""
+    OpenAPI Petstore
+
+    This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\  # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import sys
+import unittest
+import json
+from collections import namedtuple
+
+
+import petstore_api
+try:
+    from petstore_api.model import (
+        animal,
+        dog,
+    )
+    from petstore_api.api_client import ApiClient
+except ImportError:
+    animal = sys.modules[
+        'petstore_api.model.animal']
+    dog = sys.modules[
+        'petstore_api.model.dog']
```

Comment:
How about doing:
```
import petstore_api
from petstore_api.model.dog import Dog
from petstore_api.model.animal import Animal
```
There should not be an import error when importing models directly.
Note: ApiClient is already accessible at petstore_api.ApiClient
Verification:
```
(venv) Justins-MacBook-Air:python justinblack$ python
Python 3.8.10 (default, May  4 2021, 03:04:19) 
>>> import petstore_api
>>> from petstore_api.model.dog import Dog
>>> from petstore_api.model.animal import Animal
>>> petstore_api.ApiClient
<class 'petstore_api.api_client.ApiClient'>
```
No import errors